types/subleveldown/index.d.ts

```diff
@@ -0,0 +1,33 @@
+// Type definitions for subleveldown 4.1
+// Project: https://github.com/level/subleveldown
+// Definitions by: Carson Farmer <https://github.com/carsonfarmer>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LevelUp } from 'levelup';
+import { ErrorValueCallback, ErrorCallback, AbstractGetOptions } from 'abstract-leveldown';
+
+declare namespace sub {
+    interface Options {
+        separator?: string;
+        open?: ((callback?: ErrorCallback) => any);
+        [key: string]: any;
```

Comment:
The source code uses only [separator] and [open]. Why is it here?

[separator]: https://github.com/Level/subleveldown/blob/master/leveldown.js#L51

[open]: https://github.com/Level/subleveldown/blob/master/leveldown.js#L62

Reply:
That was a relic of some old work, thanks for catching! Fixed.